mathicsscript/__init__.py

```diff
@@ -5,3 +5,15 @@
 Copyright 2020-2021 The Mathics Team
 """
 from mathicsscript.version import __version__
+
+
+__all__ = [__version__]
+
+
+def load_default_settings_files(definitions):
```

Comment:
When `__all__` is used, things you want exported like `load_default_settings_files()` I think are supposed to be included inside the `__all__` list.

So 

```python
def load_default_settings_files(definitions):
  ...

___all__  = [__version__, load_default_settings_files]

Reply:
Ok.